docs/community/index.md

```diff
@@ -0,0 +1,15 @@
+## Welcome
+Welcome to physiopy! We are an international community of volunteers that meet monthly to discuss methods development of our tools and packages and to host discussions of best practices for physiological data acquisition, preprocessing, and analysis. We are always looking for new members and contributors so if interested, please reach out!
+
+### When do we meet?
+We meet virtually on zoom the third Thursday of every month at 16:00 UTC. To join please email physiopy@gmail.com for the link.
+
+### How to get involved?
+To receive emails about our monthly meetings and planning surveys for topics to be covered please email physiopy@gmail.com and join our mattermost channel or check out our [gitter](https://gitter.im/physiopy/community) for the most up-to-date information.
+
+### Citations
+When using physiopy, please include the following citations:
+
+- Daniel Alcalá, Apoorva Ayyagari, Katie Bottenhorn, Molly Bright, César Caballero-Gaudes, Inés Chavarría, Vicente Ferrer, Soichi Hayashi, Vittorio Iacovella, François Lespinasse, Ross Markello, Stefano Moia, Robert Oostenveld, Taylor Salo, Rachael Stickland, Eneko Uruñuela, Merel van der Thiel, & Kristina Zvolanek. (2021). physiopy/phys2bids: BIDS formatting of physiological recordings (2.6.0). Zenodo. https://doi.org/10.5281/zenodo.4983522
+    - [pysiopy/phys2bids: BIDS formatting of physiological recordings](https://zenodo.org/record/4983522#.Yq2v9-zMLPY)
+- [Physiopy: A community-driven suite of tools for physiological recordings in neuroimaging](https://www.ismrm.org/21/program-files/TeaserSlides/TeasersPresentations/0726-Teaser.html)<FIX THIS REFERENCE>

```

Comment:
```suggestion
```

I'd honestly remove citations from here - all packages have their own citations that we can link from the main page of the documentation, and documentation will have its own citation!